### ISSUE TITLE
visible label and accessible name do not match

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3929]
The button to dismiss the cookie message displays the text 'close' but has an arial-label of 'dismiss cookie message" which do not match.

The aria labels are not needed when the link is clear from the context (hence using text other than 'click here' for links).  By having both aria label and text, and them being different, this makes the link confusing.

**Comment by ajrbyers** [comment]
@StephDriver to find out where this error shows

**Comment by StephDriver** [comment]
example:

```html
<a aria-label="learn more about cookies" role="button" tabindex="0" class="cc-link" rel="noopener noreferrer nofollow" target="_blank" href="/cookies">use of cookies.</a>
```

"learn more about cookies" doesn't mean quite the same thing as "use of cookies"

this should be "learn more about our use of cookies"

**Comment by joemull** [comment]
I've had a cursory look at this and I don't think it's our code. Closing this and referring it back to Iowa.
